wmtsCacheServer/apiutils.py

```diff
@@ -64,10 +64,25 @@ def prepare(self, **values):
         pass
 
     def href(self, path: str="", extension: str="") -> str:
-        """ Returns an URL to self, to be used for links to the current resources 
+        """ Returns an URL to self, to be used for links to the current resources
             and as a base for constructing links to sub-resources
         """
-        return self._parent.href(self._context,path,extension)
+        return self._parent.href(self._context, path, extension)
+
+    def links(self) -> List[str]:
```

Comment:
There is no point to implement this here, apiutils is not a global library and will never be one because of the way Qgis plugin are distributed (with no dependencies)